src/vivarium_conic_lsff/components/observers.py

```diff
@@ -132,4 +133,95 @@ def get_transition_count(pop, config, disease, transition, event_time, age_bins)
                                                         year=event_time.year)
     base_filter = QueryString('')
     transition_count = get_group_counts(transitioned_pop, base_filter, base_key, config, age_bins)
-    return transition_count
\ No newline at end of file
+    return transition_count
+
+
+
+class LiveBirthWithNTDObserver:
+    """Observes births and births with neural tube defects. Output can be stratified
+    by year and by sex.
+    """
+    configuration_defaults = {
+        'metrics': {
+            project_globals.NTD_OBSERVER: {
+                'by_year': True,
+                'by_sex': True,
+            }
+        }
+    }
+
+    @property
+    def name(self):
+        return project_globals.NTD_OBSERVER
+
+    def setup(self, builder):
+        self.disease = project_globals.NTD_MODEL_NAME
+        self.config = builder.configuration['metrics'][project_globals.NTD_OBSERVER].to_dict()
+        self.config['by_age'] = False
+
+        tmp = builder.configuration['time']['start']
+        self._sim_start = pd.Timestamp(tmp.year, tmp.month, tmp.day)
+        tmp = builder.configuration['time']['end']
+        self._sim_end = pd.Timestamp(tmp.year, tmp.month, tmp.day)
+
+        columns_required = ['alive', 'dead', f'{self.disease}', 'entrance_time', 'tracked', 'untracked']
+        if self.config['by_sex']:
+            columns_required.append('sex')
+
+        self.population_view = builder.population.get_view(columns_required)
+        builder.value.register_value_modifier('metrics', self.metrics)
+
+
+    def metrics(self, index, metrics):
+        pop = self.population_view.get(index)
+        births = get_births(pop, self.config, self._sim_start, self._sim_end)
+        metrics.update(births)
+        return metrics
+
+
+    def __repr__(self):
+        return f"DiseaseObserver({self.disease})"
+
+
+def get_births(pop: pd.DataFrame, config: Dict[str, bool], sim_start: pd.Timestamp,
+               sim_end: pd.Timestamp) -> Dict[str, int]:
+    """Counts the number of births and births with neural tube defects prevelant.
+    Parameters
+    ----------
+    pop
+        The population dataframe to be counted. It must contain sufficient
+        columns for any necessary filtering (e.g. the ``age`` column if
+        filtering by age).
+    config
+        A dict with ``by_age``, ``by_sex``, and ``by_year`` keys and
+        boolean values.
+    sim_start
+        The simulation start time.
+    sim_end
+        The simulation end time.
+    Returns
+    -------
+    births
+        All births and births with neural tube defects present.
+    """
+    base_filter = QueryString('')
+    base_key = get_output_template(**config)
+    time_spans = get_time_iterable(config, sim_start, sim_end)
+
+    births = {}
+    for year, (t_start, t_end) in time_spans:
+        start = max(sim_start, t_start)
+        end = min(sim_end, t_end)
+        born_in_span = pop.query(f'entrance_time >= "{start}" and entrance_time < "{end}"')
```

Comment:
Readability thing:  I think it's a good idea to pick exactly one ordering operator and always use it. I typically only use `<`.  To me `'{start} <= entrance_time and entrance_time < {end}' ` is much clearer.